sdk/identity/src/token_credentials/imds_managed_identity_credentials.rs

```diff
@@ -18,7 +18,7 @@ const MSI_API_VERSION: &str = "2019-08-01";
 /// This authentication type works in Azure VMs, App Service and Azure Functions applications, as well as the Azure Cloud Shell
 ///
 /// Built up from docs at [https://docs.microsoft.com/azure/app-service/overview-managed-identity#using-the-rest-protocol](https://docs.microsoft.com/azure/app-service/overview-managed-identity#using-the-rest-protocol)
-pub struct ManagedIdentityCredential;
+pub struct ImdsManagedIdentityCredential;
```

Comment:
Just adding a note here since this PR didn't actually change this, but we may want to make this a sturct with a private field to allow us to add fields in the future:

```rust
pub struct ImdsManagedIdentityCredential {
  _priv: ()
}
```